build.gradle

```diff
@@ -265,7 +265,7 @@ dependencies {
         runtimeOnly 'org.apache.ivy:ivy:2.5.1'
 
         // Javascript used for scripts
-        implementation 'org.openjdk.nashorn:nashorn-core:15.4'
+        runtimeOnly 'org.openjdk.nashorn:nashorn-core:15.4'
```

Comment:
Required for compilation of deprecated `JsonParser` class on Java 17 and later.

Reply:
Because OmegaT 6.1 is complied with Java 11, so it can be `runtimeOnly`.
There is a pending discussion for future version when we remove deprecated `o.o.u.JsonParser` class.